util/SDIF/CMakeLists.txt

```diff
@@ -3,4 +3,7 @@
 if(BUILD_UTILITIES)
 set(sdif2ad_SRCS sdif2adsyn.c sdif.c sdif-mem.c)
 make_utility(sdif2ad "${sdif2ad_SRCS}" "")
+if(${CMAKE_BUILD_TYPE} MATCHES "Release")
+    target_compile_definitions(sdif2ad PRIVATE NDEBUG) # needed for asserts
+endif()

```

Comment:
Why not just add this into `make_utility` and have it uniform across utilities?

Reply:
`sdif2ad` is the only utility or plugin with asserts